starboard/android/shared/player_components_factory.h

```diff
@@ -80,6 +80,11 @@ constexpr bool kForceFlushDecoderDuringReset = false;
 // during Reset(). This should be enabled with kForceFlushDecoderDuringReset.
 constexpr bool kForceResetAudioDecoder = false;
 
+// Use a new created handler thread to handle mediacodec callbacks. This
+// prevents the mediacodec callbacks are blocked by activity lifecycle
+// callbacks.
+constexpr bool kUseHandlerThread = false;
```

Comment:
Consider to rename this to `kUseMediaCodecCallbackThread` to align with the `mediacodeccallbackthread`.

Reply:
revised